src/main/java/uk/gov/digital/ho/hocs/domain/queue/migration/MigrationService.java

```diff
@@ -185,15 +189,19 @@ void createCaseAttachments(String messageId, UUID caseId, MigrationData migratio
     }
 
     CreateMigrationCaseRequest composeMigrateCaseRequest(
+        String messageId,
         MigrationData migrationData,
         MigrationCaseTypeData migrationCaseTypeData
     ) {
-        Map<String, String> initialData = Map.of(CHANNEL_LABEL, migrationCaseTypeData.getOrigin());
+        Map<String, String> initialData =
+            caseDataService.parseCaseDataJson(messageId, migrationData.getCaseDataJson());
+        initialData.put(CHANNEL_LABEL, migrationCaseTypeData.getOrigin());
```

Comment:
I think this 'Channel' item can be removed. It should be included from the LHS. It's 'Channel' for COMP, 'ComplaintChannel' for POGR, and doesn't exist in the workflow for IEDET or BF. Unless it's red herring-ing me and so core hidden field or something.

Reply:
This is a field specific to cases ingested via case creator. It is either "Webform" or "CMS" depending on the source.

Reply:
Correction - I can see it being collected in workflow for normal cases. And case creator logic does seem to be written assuming it'll be a COMP case.

Unless the LHS is providing a value for this should it be set to CMS for closed cases where there isn't user interaction to set this to the correct value? It won't cause any issues if it's set for case types that don't use it. Happy to remove if that's preferred though.